business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml

```diff
@@ -12,6 +12,36 @@
                     <field name="categ_id" 
                            attrs="{'readonly':[('resource_type','=','procurement')],'required':[('resource_type','=','task')]}"/>
                 </field>
+            </field>business_requirement_deliverable_project_task_categ
+        </record>
+
+        <!-- Search View Business Requirement Resource Line-->
+        <record id="view_business_requirement_resource_search" model="ir.ui.view">
+            <field name="name">business.requirement.resource.search</field>
+            <field name="model">business.requirement.resource</field>
+            <field name="inherit_id"
+                   ref="business_requirement_deliverable.view_business_requirement_resource_search"/>
+            <field name="arch" type="xml">
+                <field name="product_id" position="after">
+                    <field name="categ_id"/>
+                </field>
+                <group expand="0" position="inside">
+                    <filter string="Category" icon="terp-go-home" context="{'group_by':'categ_id'}"/>
+                </group>
+            </field>
+        </record>
+
+        <!-- Requirement Deliverable Line Noedit form view-->
+        <record id="view_business_requirement_deliverable_form" model="ir.ui.view">
+            <field name="name">business.requirement.deliverable.form.noedit</field>
+            <field name="model">business.requirement.deliverable</field>
+            <field name="inherit_id"
+                   ref="business_requirement_deliverable.view_business_requirement_deliverable_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='resource_ids']/tree[@editable='1']/field[@name='name']"
```

Comment:
Is the `[@editable='1']` really needed? Can we get away with a more flexible query, such as `tree[0]`?